odoo/addons/base/tests/test_translate.py

```diff
@@ -646,29 +646,41 @@ def test_sync(self):
         terms_en = ('Bread and cheeze',)
         terms_fr = ('Pain et fromage',)
         terms_nl = ('Brood and kaas',)
```

Comment:
Who said Walloons are not able to speak Dutch? :roll_eyes:

Reply:
it was probably to test that if we remove 3 of 14 letters the string is not close enough :see_no_evil: